<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainMenu</class>
 <widget class="QWidget" name="mainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2006</width>
    <height>1141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mainPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignHCenter">
        <widget class="QFrame" name="mainMenu_2">
         <property name="minimumSize">
          <size>
           <width>461</width>
           <height>361</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>461</width>
           <height>361</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rbg(170, 169, 173)</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="createProgression">
            <property name="minimumSize">
             <size>
              <width>361</width>
              <height>41</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>361</width>
              <height>41</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rbg(225, 224, 228)</string>
            </property>
            <property name="text">
             <string>Create Progression</string>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="loadProgression">
            <property name="minimumSize">
             <size>
              <width>361</width>
              <height>41</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>361</width>
              <height>41</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rbg(225, 224, 228)</string>
            </property>
            <property name="text">
             <string>Load Progression</string>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="openPacks">
            <property name="minimumSize">
             <size>
              <width>361</width>
              <height>41</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>361</width>
              <height>41</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rbg(225, 224, 228)</string>
            </property>
            <property name="text">
             <string>Open Packs</string>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="editDecks">
            <property name="minimumSize">
             <size>
              <width>361</width>
              <height>41</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>361</width>
              <height>41</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rbg(225, 224, 228)</string>
            </property>
            <property name="text">
             <string>Edit Decks</string>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="packOpener">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QFrame" name="frame">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(48, 255, 16);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
